### ISSUE TITLE
Singular exceptions for suspicious code rejection

### ISSUE THREAD

**Issue by ThePeashooter23** [studio-minus/ppg-bugs-and-requests#1557]
**Feature description**
Ability to turn off suspicious code rejection on one mod. Instead of just having a global on/off in the settings. 

**Feature purpose**
There are some trustworthy mods that need features that would normally be "suspicious code". Instead of risking having other potentially malicious mods get activated, trusting just one mod would be safer in my opinion.

**Additional context**
I was inspired to write this after trying to install "Parts and Contraptions" on the workshop, I have a lot of mods and got worried after being told to turn off "suspicious code rejection" The mod itself isn't malicious (it just needed mp3 files from the internet.) but I was iffy about disabling the setting.

**Comment by PossiblyRemember** [comment]
I totally agree with this, I didn't get the mod because it required shady mods to be enabled which really makes me feel unsafe.
